.travis.yml

```diff
@@ -70,6 +70,11 @@ jobs:
 stages:
   - format_analyzer_tests
 
+os:
```

Comment:
This file is generated, the update should happen in mono_repo.yml

Reply:
New attempt in https://github.com/dart-lang/protobuf/pull/238